### DISCUSSION TITLE
Is it possible to extract fonts?

### DISCUSSION THREAD

**Discussion by antar4** [UglyToad/PdfPig#487] [Q&A]
Is it possible to extract fonts? I can see the 'Letter' being created with only the FontDetails and no way back to the IFont itself. Also I can't see anyway to access the resourceStore that has the actual fonts. Am I missing something?

**Comment by fnatzke** [comment]
Extracting font(s) from a PDF is not a technical problem but ..... 

1) Your might be amazed that most fonts have a copyright. Extracting font data from a PDF into a "font file" would be illegal.  
2) An embedded font is not alway complete (to save space) for example containing only a subset of the letters enough to render the letters within the document (eg. a,e,i,o,u and not w,x,y or z)
 
Outside "extracting font data to a file" (due to problems above) how else would you like to use IFont (and classes that implement the interface eg. TrueTypeSimpleFont, Type0Font, ...)?
